db/index.js

```diff
@@ -63,12 +65,17 @@ function insert_transaction(
   category_id,
   vendor,
   amount,
+  file_name,
   callback
 ) {
+  const fs = require('fs');
```

Comment:
requires should go at the top of files

Reply:
Also, is FS built into node by default or does this need to be added to package.json?